### ISSUE TITLE
Dúvida 

### ISSUE THREAD

**Issue by jurgenjacobsen** [SmartBR/discord-bot-web#1]
Muito obrigado pela este repositório, gostaria de saber se teria como eu alterar a cor da borda dos botões. Tem alguma forma ?

**Comment by SmartBR** [comment]
Sim, os botões estão localizados em "src/website/views/index.pug". 

O padrão está a.btn.btn-outline-warning(href="/commands"), você pode mudar a palavra warning para uma dessas abaixo:
primary - azul
secondary - roxo
success - verde claro
warning - Amarelo, meio laranjado
info - um tipo de ciano, só que mais escuro
danger - vermelho
light - branco
dark - preto

Ou se preferir, pode mudar utilizando o próprio CSS.
